taipy/gui/viselements.json

```diff
@@ -1113,6 +1113,11 @@
             "type": "dynamic(bool)",
             "default_value": "False",
             "doc": "If set to True, this allows to dynamically refresh the  columns."
+          },
+          {
+            "name": "lov[<i>column_name</i>]",
+            "type": "list[str]|str",
+            "doc": "The list of values of the indicated column."
```

Comment:
```suggestion
            "doc": "The list of valid values for the indicated column. It can be defined as a series of comma separated strings."
```

Reply:
That's the behavior: a list of values but you can type your own

Reply:
adding support for None as an indicator that the user can enter any value